src/environment_setup/biopam.mli

```diff
@@ -8,23 +8,22 @@ val default_opam_url : string
 (** The default location from where we download biopam. *)
 val default_biopam_url : string
 
-(** A workflow to make sure that Biopam is configured.*)
-val configured : ?biopam_home:string -> ?host:KEDSL.Host.t ->
-  install_path:string -> unit ->
-  < is_done : KEDSL.Condition.t option > KEDSL.workflow_node
-
+type tool_type = [
+  | `Library of string   (** The export variable that points to witness. *)
```

Comment:
Yes that's what I meant by “hijacking” the semantics; it's not clear and could be package dependent. I keep that for a later PR.